archiver.go

```diff
@@ -121,6 +121,11 @@ func (a *Archiver) do(log log15.Logger, j *Job) (err error) {
 		)
 
 		metrics.RepoSkipped()
+		r.FetchErrorAt = &now
```

Comment:
if the repo is marked as failed, then it does not make sense to have `metrics.RepoSkipped` called because it was not skipped, it was failed